packages/wrangler/pages/functions/buildWorker.ts

```diff
@@ -1,11 +1,13 @@
 import path from "path";
```

Comment:
maybe it's a good time to make a build of this file too. I'd like to keep packages out of dependencies, and would rather deliver bundled files. In a followup PR is fine, but before GA.